src/utils/validateName.ts

```diff
@@ -0,0 +1,7 @@
+// prettier-ignore
+export default function validateName(name: string): string | void {
+    if (!name.match(/[0-9]/g)) return "Name must include at least 1 number";
```

Comment:
This will only match part of the name, and will allow other non-wanted characters, as @AhmedHanafy725 mentioned, it need to be an alpha-numeric with max length of 15.